comps/comps-foreman-fedora24.xml

```diff
@@ -29,13 +29,15 @@
       <packagereq type="default">foreman-selinux</packagereq>
       <packagereq type="default">foreman-sqlite</packagereq>
       <packagereq type="default">foreman-vmware</packagereq>
+      <packagereq type="default">nodejs-async</packagereq>
```

Comment:
nodejs-async 1.5's already in the distribution, is the older dep going to resolve correctly, since it's `0.2.x`? Bundling the dependency would be preferable I think in this case.